etcdctl/READMEv3.md

```diff
@@ -735,6 +735,59 @@ cf1550fb, 3, 3, 25 kB
 +----------+----------+------------+------------+
 ```
 
+### MIGRATE [options]
+
+Migrate migrates keys in store v2 into mvcc store.
+
+#### Options
+
+- data-dir -- Path to the data directory
+
+- wal-dir -- Path to the WAL directory
+
+- transformer -- Path to the user-provided transformer program (default if not provided)
+
+#### Return value
+
+Simple reply
+
+- Exit code is zero when migration is finished successfully.
+
+- Error string if migration failed. Exit code is non-zero.
+
+#### Default transformer
+
+If user does not provide a transformer program, migrate command will use the default transformer. The default transformer transforms keys in store v2 into mvcc store according to the following Go program:
```

Comment:
The default transformer transforms `storev2` formatted keys into `mvcc` formatted keys according to the following Go program: